cmd/kubelet/app/options/globalflags.go

```diff
@@ -17,94 +17,31 @@ limitations under the License.
 package options
 
 import (
-	"flag"
-	"fmt"
-	"os"
-	"strings"
-
 	"github.com/spf13/pflag"
 
 	// libs that provide registration functions
-	"k8s.io/component-base/logs"
+	"k8s.io/component-base/cli/globalflag"
 	"k8s.io/kubernetes/pkg/version/verflag"
 
 	// ensure libs have a chance to globally register their flags
-	_ "k8s.io/klog"
 	_ "k8s.io/kubernetes/pkg/credentialprovider/azure"
 	_ "k8s.io/kubernetes/pkg/credentialprovider/gcp"
 )
 
-// AddGlobalFlags explicitly registers flags that libraries (glog, verflag, etc.) register
+// AddCustomGlobalFlags explicitly registers flags that libraries (glog, verflag, etc.) register
 // against the global flagsets from "flag" and "github.com/spf13/pflag".
 // We do this in order to prevent unwanted flags from leaking into the Kubelet's flagset.
-func AddGlobalFlags(fs *pflag.FlagSet) {
-	addGlogFlags(fs)
```

Comment:
I'm not sure that the component-base globalflags package should actually register any flags, vs. just providing the helpers.

Reply:
run `bash hack/local-up-cluster.sh` in local ENV,  from run parameter at the begining of log `/tmp/kube-controller-manager.log`  and `/tmp/kubelet.log`, we can see 
```
I0414 15:47:16.050073   29547 flags.go:33] FLAG: --tls-private-key-file=""
I0414 15:47:16.050077   29547 flags.go:33] FLAG: --v="3"
I0414 15:47:16.050080   29547 flags.go:33] FLAG: --version="false"
```
`-v` in globalFlag set, registed, value is `3`